identity/src/commonMain/kotlin/com/android/identity/nfc/ServiceParameterRecord.kt

```diff
@@ -0,0 +1,88 @@
+package com.android.identity.nfc
+
+import kotlinx.io.bytestring.ByteStringBuilder
+import kotlin.math.pow
+import kotlin.time.Duration
+
+/**
+ * Service Parameter Record.
+ *
+ * Reference: NFC Forum Tag NDEF Exchange Protocol section 4.1.2 Service Parameter Record.
+ *
+ * @property tnepVersion TNEP version.
+ * @property serviceNameUri Service Name URI.
+ * @property tnepCommunicationMode TNEP communication mode.
+ * @property wtInt Minimum waiting time, use [Duration.Companion.fromWtInt] to convert to a [Duration].
+ * @property nWait Maximum number of waiting time extensions.
+ * @property maxNdefSize Maximum NDEF Message size in bytes.
+ */
+data class ServiceParameterRecord(
+    val tnepVersion: Int,
+    val serviceNameUri: String,
+    val tnepCommunicationMode: Int,
+    val wtInt: Int,
```

Comment:
Similar question regarding wtInt as a Duration...and naming...

Reply:
Yeah, same answer.